### DISCUSSION TITLE
Roles

### DISCUSSION THREAD

**Discussion by LeliBRUK** [Tandashi/YTSync-Plugin#44] [Ideas]
It would be cool if you as a host could customise roles and their permissions (just like discord?)

**Comment by Tandashi** [comment]
In the current state of the Extension I don't see a real need for it since the 3 pre-existing roles should cover pretty-much every situation. 

Do you have some examples of roles and their permissions you would like to create with that? 
Also keep in mind that rooms are not permanent. They vanish after everyone has left, therefor one would need to recreate / recustomize every role each time.
